webtool/test_jobs.py

```
@@ -459,6 +459,82 @@ def test_done_entfernt_aufnahme_aus_active_bases():
+    try:
+        _warte_auf_zeilen(jid, 2)
+        assert jobs.get(jid)["gesehen"] == []
+        # Gegenprobe: dieselbe Zeile, andere Art -> die Buchung findet statt. Ohne sie bliebe
+        # der Test gruen, wenn `gesehen` aus einem ganz anderen Grund nie gefuellt wuerde.
+        jid2, _ = jobs.start("P_correct_gesehen", [sys.executable, "-c", code], cwd=None,
+                             kind="correct")
+        try:
+            _warte_auf_zeilen(jid2, 2)
+            assert jobs.get(jid2)["gesehen"] == ["Fremdtitel"]
+        finally:
+            jobs.cancel(jid2); _wait(jid2)
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)